web app/src/main/java/de/tinf15b4/kino/data/cinemas/Cinema.java

```diff
@@ -23,6 +23,21 @@
     @Column(length = 100000)
     private byte[] image;
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof Cinema)) return false;
+
+        Cinema cinema = (Cinema) o;
+
+        return id == cinema.id;
+    }
+
+    @Override
+    public int hashCode() {
+        return (int) (id ^ (id >>> 32));
```

Comment:
Yes. This is what IntelliJ outputs when you make it generate equals and hashcode over the id field. (Interesting side question: What does eclipse generate in such a case? Just cast it to an int?)